index.html

```diff
@@ -58,8 +63,13 @@ <h3 class="label">Photo URL</h3>
             <div class="row justify-end">
               <input type="submit" value="SAVE" class="save">
             </div>
+          </form>
+        </div>
+        <div data-view="entries" class="entries hidden">
+        </div>
```

Comment:
Either your indentation is messed up or this element is in the wrong spot. Either way, please revise